flutter_gallery__back_button_memory is flaky

We need to fix the flakiness or delete the test if it is no longer relevant.

Broken out from #58338 /cc @Hixie

Strange, it looks like the failure has something to do with the LatinIME using `java.nio.file.Paths`, and that not existing in the classpath.

<details>
<pre>
2020-06-03T22:16:31.229969: adb logcat: 06-03 22:15:55.137 20701 20847 I ActivityManager: Start proc 22793:com.google.android.inputmethod.latin/u0a94 for service com.google.android.inputmethod.latin/com.android.inputmethod.latin.LatinIME
2020-06-03T22:16:31.890981: adb logcat: --------- beginning of crash
2020-06-03T22:16:31.900142: stdout: [ +741 ms] E/AndroidRuntime(22793): FATAL EXCEPTION: main
2020-06-03T22:16:31.900748: stdout: [        ] E/AndroidRuntime(22793): Process: com.google.android.inputmethod.latin, PID: 22793
2020-06-03T22:16:31.901308: stdout: [        ] E/AndroidRuntime(22793): java.lang.NoClassDefFoundError: Failed resolution of: Ljava/nio/file/Paths;
2020-06-03T22:16:31.901933: stdout: [        ] E/AndroidRuntime(22793): 	at efg.<init>(PG:4)
2020-06-03T22:16:31.902436: stdout: [        ] E/AndroidRuntime(22793): 	at eem.a(PG:4)
2020-06-03T22:16:31.903235: stdout: [        ] E/AndroidRuntime(22793): 	at efb.<init>(PG:9)
2020-06-03T22:16:31.903883: stdout: [        ] E/AndroidRuntime(22793): 	at eem.<init>(PG:1)
2020-06-03T22:16:31.904375: stdout: [        ] E/AndroidRuntime(22793): 	at efc.<init>(PG:10)
2020-06-03T22:16:31.905031: stdout: [        ] E/AndroidRuntime(22793): 	at efc.a(PG:16)
2020-06-03T22:16:31.905480: stdout: [        ] E/AndroidRuntime(22793): 	at com.google.android.apps.inputmethod.libs.handwriting.ime.HandwritingImeExtension.a(PG:4)
2020-06-03T22:16:31.906287: stdout: [        ] E/AndroidRuntime(22793): 	at khy.b(PG:15)
2020-06-03T22:16:31.906969: stdout: [        ] E/AndroidRuntime(22793): 	at khy.a(PG:29)
2020-06-03T22:16:31.909903: stdout: [        ] E/AndroidRuntime(22793): 	at kih.a(PG:100)
stdout: [        ] E/AndroidRuntime(22793): 	at kig.b(PG:10)
stdout: [        ] E/AndroidRuntime(22793): 	at kia.run(PG:2)
stdout: [        ] E/AndroidRuntime(22793): 	at kjt.d(PG:7)
2020-06-03T22:16:31.910443: stdout: [        ] E/AndroidRuntime(22793): 	at kjw.b(PG:8)
2020-06-03T22:16:31.910865: stdout: [        ] E/AndroidRuntime(22793): 	at kjs.a(PG:10)
2020-06-03T22:16:31.911309: stdout: [        ] E/AndroidRuntime(22793): 	at kjs.a(PG:11)
2020-06-03T22:16:31.912247: stdout: [        ] E/AndroidRuntime(22793): 	at kke.run(PG:7)
2020-06-03T22:16:31.912672: stdout: [        ] E/AndroidRuntime(22793): 	at android.os.Handler.handleCallback(Handler.java:751)
2020-06-03T22:16:31.914273: stdout: [        ] E/AndroidRuntime(22793): 	at android.os.Handler.dispatchMessage(Handler.java:95)
stdout: [        ] E/AndroidRuntime(22793): 	at android.os.Looper.loop(Looper.java:154)
2020-06-03T22:16:31.914468: stdout: [        ] E/AndroidRuntime(22793): 	at android.app.ActivityThread.main(ActivityThread.java:6123)
2020-06-03T22:16:31.915488: stdout: [        ] E/AndroidRuntime(22793): 	at java.lang.reflect.Method.invoke(Native Method)
stdout: [        ] E/AndroidRuntime(22793): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:867)
2020-06-03T22:16:31.916391: stdout: [        ] E/AndroidRuntime(22793): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:757)
2020-06-03T22:16:31.916713: stdout: [        ] E/AndroidRuntime(22793): Caused by: java.lang.ClassNotFoundException: Didn't find class "java.nio.file.Paths" on path: DexPathList[[zip file "/data/app/com.google.android.inputmethod.latin-1/base.apk"],nativeLibraryDirectories=[/data/app/com.google.android.inputmethod.latin-1/lib/arm, /data/app/com.google.android.inputmethod.latin-1/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib]]
2020-06-03T22:16:31.917555: stdout: [        ] E/AndroidRuntime(22793): 	at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
2020-06-03T22:16:31.917900: stdout: [        ] E/AndroidRuntime(22793): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
2020-06-03T22:16:31.918617: stdout: [        ] E/AndroidRuntime(22793): 	at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
2020-06-03T22:16:31.918913: stdout: [        ] E/AndroidRuntime(22793): 	... 24 more
</pre>
</details>

--- a/dev/devicelab/manifest.yaml
+++ b/dev/devicelab/manifest.yaml
@@ -744,14 +744,12 @@ tasks:
       Measures memory usage after repeated navigation in Gallery.
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
-    flaky: true
 
   flutter_gallery__back_button_memory:
     description: >
       Measures memory usage after Android app suspend and resume.
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
-    flaky: true
 
   flutter_gallery__image_cache_memory:
     description: >

